content/v2.0/query-data/flux/sql.md

```diff
@@ -154,6 +155,78 @@ Use the variable to manipulate queries in your dashboards.
 
 ---
 
+## Use secrets to populate connection credentials
+If your SQL database requires authentication, use [InfluxDB secrets](/v2.0/security/secrets/)
+to store and populate connection credentials.
+By default, InfluxDB base64-encodes and stores secrets in its internal key-value store, BoltDB.
+For added security, [Vault](https://www.vaultproject.io/).
+
+_For information about the **InfluxDB Vault integration**, see [Store secrets in Vault](/v2.0/security/secrets/use-vault/)._
```

Comment:
For added security, [store secrets in Vault](/v2.0/security/secrets/use-vault/).